frontend/src/app/app.routing.ts

```diff
@@ -5,42 +5,12 @@ import { PageNotFoundComponent } from './page-not-found/page-not-found.component
 import { ChartIndexComponent } from './chart-index/chart-index.component';
 import { ChartDetailsComponent } from './chart-details/chart-details.component';
 import { ChartsComponent } from './charts/charts.component';
-import { DeploymentsComponent } from './deployments/deployments.component';
-import { DeploymentComponent } from './deployment/deployment.component';
-import { RepositoriesComponent } from './repositories/repositories.component';
 
 const appRoutes: Routes = [
   {
     path: '',
     component: ChartIndexComponent
   },
-  {
-    path: 'deployments',
-    component: DeploymentsComponent,
-    data: {
-      meta: {
-        title: 'Current deployments'
-      }
-    }
-  },
-  {
-    path: 'deployments/:deploymentName',
-    component: DeploymentComponent,
-    data: {
-      meta: {
-        title: 'Deployment information'
-      }
-    }
-  },
-  {
-    path: 'repos',
```

Comment:
How is repositories related to releases?

Reply:
This page was added for an in-cluster feature to add and delete synced repos, it is not used in public site usecases such as https://hub.kubeapps.com. I believe this will not be that useful to the CNCF Helm Hub, but we can always revist it. The list of repos to browse can be seen on the charts page, e.g. https://hub.kubeapps.com/charts